src/pages/dashboardContainer/index.jsx

```diff
@@ -1,16 +1,9 @@
 import React, { useEffect, useState, useRef } from 'react';
 import { useSearchParams } from 'react-router-dom';
-
-import { Dashboard } from '../dashboard/index.jsx';
-import { fetchAllFromSTACAPI } from '../../services/api';
-import {
-  dataTransformationPlume,
-  dataTransformationPlumeRegion,
-  dataTransformationPlumeMeta,
-  dataTransformationPlumeRegionMeta,
-  metaDatetimeFix,
-} from './helper/dataTransform';
-import { PlumeMetas } from '../../assets/dataset/metadata.ts';
+import { EmitDashboard } from '../dashboard/index.jsx';
+import { Plumes } from '../../assets/dataset/testData.js';
```

Comment:
do not rely too much on test data.
Its okay to use mock/test data if the data is not prepared/complete.
However, we have it so use the real data source instead.